MapboxNavigation/Settings.swift

```diff
@@ -2,12 +2,12 @@ import Foundation
 
 extension UserDefaults {
     fileprivate func registerNavigationDefaults() {
-        register(defaults: [NavigationSettingsKey.muted.rawValue: false])
+        register(defaults: [NavigationSettingsKey.voiceVolume.rawValue: false])
     }
 }
 
 enum NavigationSettingsKey: String {
-    case muted = "muted"
+    case voiceVolume = "voiceVolume"
```

Comment:
`case voiceVolume`

Explicitly defining the raw string value is redundant if the case name is the same as the string value.